src/App/Pages/Accounts/LoginPageViewModel.cs

```diff
@@ -190,5 +207,15 @@ public void TogglePassword()
             entry.Focus();
             entry.CursorPosition = String.IsNullOrEmpty(MasterPassword) ? 0 : MasterPassword.Length;
         }
+
+        public async Task RemoveAccountAsync()
+        {
+            var confirmed = await _platformUtilsService.ShowDialogAsync(AppResources.RemoveAccountConfirmation,
+                AppResources.RemoveAccount, AppResources.Yes, AppResources.Cancel);
+            if (confirmed)
+            {
+                _messagingService.Send("logout");
+            }
```

Comment:
Add try...catch

Reply:
Can you clarify?  This is triggered directly by the user with the app in the foreground.

Reply:
@mportune-bw if the `ShowDialogAsync` or the `_messagingService` throw an exception the app will crash given that the `RemoveAccountAsync` is called from the event `RemoveAccount_Clicked`. So a `try...catch` is needed here or in `RemoveAccount_Clicked` to prevent the crash if any exception happen.